sdk/appcenter-codepush/src/androidTest/java/com/microsoft/appcenter/codepush/CodePushTest.java

```diff
@@ -0,0 +1,17 @@
+package com.microsoft.appcenter.codepush;
+
+import android.support.test.runner.AndroidJUnit4;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.junit.Assert.*;
+
+@RunWith(AndroidJUnit4.class)
```

Comment:
Remove this annotation, its already junit 4 by default in all our tests

Reply:
@guperrot Removed it, thanks.